Sources/SmokeAsyncHTTP1/SmokeHTTP1HandlerSelector+futureFromProviderWithInputNoOutput.swift

```diff
@@ -0,0 +1,150 @@
+// Copyright 2018-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// A copy of the License is located at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// SmokeHTTP1HandlerSelector+futureFromProviderWithInputNoOutput.swift
+// SmokeAsyncHTTP1
+//
+
+import Foundation
+import SmokeOperations
+import NIOHTTP1
+import SmokeOperationsHTTP1
+import NIO
+import SmokeAsync
+import Logging
+
+public extension SmokeHTTP1HandlerSelector {
+    /**
+     Adds a handler for the specified uri and http method.
+ 
+     - Parameters:
+        - uri: The uri to add the handler for.
```

Comment:
nit: no uri param, can we update this?

Reply:
Thanks for catching this. Looks like this wasn't changed a while ago and has been copied to a whole bunch of places. I have fixed them as well.